### PR TITLE
Only run the 'pre' tag on PRs, not 'pr'.

### PR DESCRIPTION
**Pull request by ashleypittman**
This probably wants new tags, for example 'pr' for prs and 'landings'
for master but using the pr tag for master here means fewer changes
and potential conflicts when testing this change.

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

**Comment by brianjmurrell** [comment]
What's the goal here?

**Comment by ashleypittman** [comment]
> What's the goal here?

https://github.com/daos-stack/daos/pull/4059

Have a seperate test list for PRs as opposed to landings/master.

File: vars/parseStageInfo.groovy [hunk 1]
```diff
@@ -114,22 +114,27 @@ def call(Map config = [:]) {
     result['node_count'] = 1
 
     if (env.STAGE_NAME.contains('Functional')) {
+      if (env.BRANCH_NAME == 'master') {
+          core_tag = 'pr'
```
Comment (brianjmurrell) [inline vars/parseStageInfo.groovy:118]: Why is the `pr` tag being used for master and...

File: vars/parseStageInfo.groovy [hunk 2]
```diff
@@ -114,22 +114,27 @@ def call(Map config = [:]) {
...
+      } else {
+          core_tag = 'pre'
```
Comment (brianjmurrell) [inline vars/parseStageInfo.groovy:120]: The tag for non-master (PRs, but actually any other [release] branch) now `pre`?  What does `pre` mean?

Comment (ashleypittman) [reply vars/parseStageInfo.groovy:120]: I put this in one of the comments, the tags should be "pr", and "landing" but in order to test that I'd need to do a search-and-replace across all tests, so for now I'm using "pre" for tests that run on PRs, and "pr" for tests that run on master, just to prove the concept with making minimal changes to the src.

**Pull request closed without merging**